cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -567,19 +598,61 @@ open class Node @Inject constructor(private val project: Project) {
         if (!config.hasPath("sshd.port")) {
             sshdPort(defaultSsh)
         }
-        val configDefaults = ConfigFactory.empty()
-            .withValue("dataSourceProperties.dataSource.url", ConfigValueFactory.fromAnyRef("jdbc:h2:file:./persistence/persistence;DB_CLOSE_ON_EXIT=FALSE;WRITE_DELAY=0;LOCK_TIMEOUT=10000"))
+
+        // TODO Refactor this code to extract the information from the config file
+        val p2pHost = config.getString("p2pAddress")
+        this.p2pPort = p2pHost.substring(p2pHost.indexOf(":") + 1).toInt()
+        val rpcConfig = config.getString("rpcSettings.address")
+        this.rpcPort = rpcConfig.substring(rpcConfig.indexOf(":") + 1).toInt()
+        val rpcAdminConfig = config.getString("rpcSettings.adminAddress")
+        val rpcAdminPort = rpcAdminConfig.substring(rpcConfig.indexOf(":") + 1)
+
         val dockerConf = config
-            .withValue("p2pAddress", ConfigValueFactory.fromAnyRef("$containerName:$p2pPort"))
-            .withValue("rpcSettings.address", ConfigValueFactory.fromAnyRef("$containerName:${rpcSettings.port}"))
-            .withValue("rpcSettings.adminAddress", ConfigValueFactory.fromAnyRef("$containerName:${rpcSettings.adminPort}"))
-            .withValue("detectPublicIp", ConfigValueFactory.fromAnyRef(false))
-            .withFallback(configDefaults)
+                .withValue("dataSourceProperties.dataSource.url", ConfigValueFactory.fromAnyRef("jdbc:h2:file:./persistence/persistence;DB_CLOSE_ON_EXIT=FALSE;WRITE_DELAY=0;LOCK_TIMEOUT=10000"))
+                .withValue("p2pAddress", ConfigValueFactory.fromAnyRef("$containerName:$p2pPort"))
+                .withValue("rpcSettings.address", ConfigValueFactory.fromAnyRef("$containerName:$rpcPort"))
+                .withValue("rpcSettings.adminAddress", ConfigValueFactory.fromAnyRef("$containerName:$rpcAdminPort"))
+                .withValue("detectPublicIp", ConfigValueFactory.fromAnyRef(false))
```

Comment:
We've lost the fallback to config defaults here, which might be important.

Reply:
These are removed because are not fallback configuration anymore. We are using the h2 config to be able to run the network bootstrap procedure, then we append the default Postgres database config